lib/bios/menus.robot

```diff
@@ -507,3 +507,40 @@ Boot System Or From Connected Disk
         ${system_index}=    Get Index Of Matching Option In Menu    ${menu_construction}    ${system_name}
     END
     Press Key N Times And Enter    ${system_index}    ${ARROW_DOWN}
+
+Check Platform Sleep Type Can Be Selected
+    Power On
+    ${setup_menu}=    Enter Setup Menu Tianocore And Return Construction
+    ${dasharo_menu}=    Enter Dasharo System Features    ${setup_menu}
+    ${power_menu}=    Enter Dasharo Submenu    ${dasharo_menu}    Power Management Options
+    ${platform_sleep_type_selectable}=    Run Keyword And Return Status
+    ...    Get Option State
+    ...    ${power_menu}
+    ...    Platform sleep type
+    Set Suite Variable    ${PLATFORM_SLEEP_TYPE_SELECTABLE}    ${platform_sleep_type_selectable}
+
+Set Platform Sleep Type
+    [Arguments]    ${platform_sleep_type}
+    Power On
+    IF    '${platform_sleep_type}' == 'S0ix'
```

Comment:
I would use different for these variables.

Reply:
`${platform_sleep_type}`
Replaced by
`${platform_sleep_type_text}`
https://github.com/Dasharo/open-source-firmware-validation/pull/128/files#diff-0d4b3b4e8afbeb5a9e8e3655f8d6ecf853f6e7e246a3da142cca93bd987e402eR23